cli/module_registry.go

```diff
@@ -699,6 +699,8 @@ func readModels(path string, logger logging.Logger) ([]ModuleComponent, error) {
 	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
 	parentAddr += "/parent.sock"
 
+	os.Setenv("VALID_PARENT_SOCK", "false")
```

Comment:
Can you defer an unsetting of this environment variable here instead of unsetting in the module code? Should make sure things get cleaned up CLI-side instead of relying on the module code to clean up.